### PR TITLE
new types for npm ejson library

### PR DESCRIPTION
**Pull request by shantanubhadoria**
case 1. Add a new type definition.
- [check] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [check] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [check] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

### CODE REVIEW

**Comment by vvakame** [comment]
http://definitelytyped.org/guides/best-practices.html#ghost-modules
please check this practive

File: ejson/ejson.d.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// Type definitions for ejson v2.1.2
+// Project: https://www.npmjs.com/package/ejson
+// Definitions by: Shantanu Bhadoria <https://github.com/shantanubhadoria>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare interface IStringifyOptions {
```
Comment (vvakame) [inline ejson/ejson.d.ts]: Could you remove `I` prefix?

Comment (shantanubhadoria) [inline ejson/ejson.d.ts]: Got it! Let me make the changes and get back to you. tslint seems to force the "I" prefix in its default configuration. Is there a tslint configuration that you use if you don't mind? 
Thanks!

**Comment by shantanubhadoria** [comment]
I have made the changes. Please comment if there is anything else.
Thanks!

**Pull request merged**